metrics/mixin/config.libsonnet

```diff
@@ -13,6 +13,8 @@
     odfPoolMirroringImageHealthCriticalAlertTime: '10s',
     blockedRBDClientAlertTime: '10s',
     ocsStorageClusterKMSConnectionAlert: '5s',
+    mdsSelector: 'pod=~"rook-ceph-mds.*"',
+    mds_cpu_usage_high_threshold_duration: '6h',
```

Comment:
This means alert will be raised only when CPU usage is high for 6 hours. Is that the desired behavior?

Reply:
yep.... seems a little bit weird to leave a pod running with more than 67% cpu usage for 6 hours.... If this happens probably the user is going to investigate by himself  why the filesystems affected are so slow before this alert will be raised.
@manishym what is your reasoning behind :
` "raise the alert when mds pod usage >= 67% for more than 6 hours"`

Reply:
Yes, by default the MDS pods are allocated 3 CPU. 67% means 2 CPUs are busy. I have given 6 hours because if there is some intermittent usage of CPU, like some scan job that reads every file or something, then they don't need multiple active MDS, if it is persistently high, then we should alert. That is the intent.

Reply:
`Sugegstion`: If that is the case we want to handle, we should probably look at something that says "if CPU usage is high most of the time then alert" instead of "if CPU usage is high for more than x hours then alert".

I still feel 6h is a long time and we should alert early. If as an admin I get this alert and I know I am running a scan job, then I can ignore it. But, we can improve it based on feedback.